eng/tox/import_all.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python
```

Comment:
Also. please run `black` on top of this new code.

1. `pip install black`
2. `black <script directory name>`

`black` is an autoformatter that rewrites your python code in a predictable fashion.

Reply:
Thanks. I have fixed the script with black and committed this change.